src/downloadlist.cpp

```diff
@@ -203,3 +208,74 @@ void DownloadList::update(int row)
   else
     log::error("invalid row {} in download list, update failed", row);
 }
+
+bool DownloadList::lessThan(const QModelIndex &left, const QModelIndex &right)

```

Comment:
Could this be named something else? I know the name is understandable but it's also the named use by the proxy model function so it took me some time to understand that this is not actually directly called and is simply used as the predicate for the filter widget.